elements/rh-cta/rh-cta.ts

```diff
@@ -2,54 +2,36 @@ import { LitElement, html } from 'lit';
 import { customElement } from 'lit/decorators/custom-element.js';
 import { property } from 'lit/decorators/property.js';
 import { classMap } from 'lit/directives/class-map.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
 
 import { Logger } from '@patternfly/pfe-core/controllers/logger.js';
 
 import { DirController } from '../../lib/DirController.js';
 
-import { type ColorPalette } from '../../lib/context/color/provider.js';
 import { colorContextConsumer, type ColorTheme } from '../../lib/context/color/consumer.js';
 
 import style from './rh-cta.css';
 
 import '@patternfly/elements/pf-icon/pf-icon.js';
```

Comment:
Can we dynamically import `pf-icon` if it's needed based on the attributes provided?

Reply:
FOUC?

Reply:
i suppose we could delay first render until icon is registered

Reply:
There's going to be FOUC no matter what with our current approach. 

Since it's already going to take longer to load if we're using pf-icon (vs the inline SVG that's there for the normal CTA styles), might as well dynamically import the JS if it's needed?

Reply:
https://github.com/RedHat-UX/red-hat-design-system/pull/1634